utils/regexToNumber.js

```diff
@@ -0,0 +1,5 @@
+
+
+export const getNumberFromString = (str) => {
```

Comment:
@karenn32 Great idea btw, but the metods name is not fully coorespond to this method. It must be named, like: "removeNumCharFromString". Well, of course not like that specifically, but the idea must be the same. Since, from the 'getNumberFromString' name someone will put it in the way like it should return the number type value.